nilearn_sandbox/datasets.py

```diff
@@ -0,0 +1,70 @@
+import os
+import cookielib
+import getpass
+
+from sklearn.datasets.base import Bunch
+
+from nilearn import datasets
+from nilearn._utils.compat import _urllib
+
+from ._utils.compat import _input
+
+
+def _get_nitrc_auth_cookie(login=None):
+    if login is None:
+        login = _input('login: ')
+    passwd = getpass.getpass('password: ')
+
+    login_url = "https://www.nitrc.org/account/login.php"
+
+    # First, we login to NITRC
+    cp = _urllib.request.HTTPCookieProcessor(cookielib.CookieJar())
+    opener = _urllib.request.build_opener(cp)
+    login_data = _urllib.parse.urlencode(
+        {'form_loginname': login, 'form_pw': passwd})
+    opener.open(login_url, login_data)
+
+    return cp
+
+
+def fetch_COBRE(data_dir=None, verbose=0):
```

Comment:
I have doc and an example script in another repo. I'll add them when I have time!